babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -431,6 +431,60 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testDateTimePrimaryColumn() {
+    final String sql = "select time_column";
```

Comment:
nit: Suggest choosing a name like "foo" or "abc" to quickly indicate to the reader that the name is arbitrary.

Reply:
Change is made.